src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,140 @@
+package duke.storage;
+
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import duke.commands.Command;
+import duke.parser.Parser;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+import duke.tasklist.TaskList;
+
+/**
+ * Class that simulates the storing and retrieving of information into the hard-disk
+ */
+public class Storage {
+    // might use it at the end
+    private static final String CREATE_FILEPATH = "Creating file path ... ... ... ... ... ... ... ... ... ...";
+    private static final String CREATE_CSV = "Creating file todoList.csv ... ... ... ... ... ... ... ...";
+    private static final String SAVE_INFO = "Saving information ... ... ... ... ... ... ... ... ... ...";
+    private static final String SAVED = "Saved  ... ... ... ... ... ... ... ... ... ... ... ... ... ";
+
+    /**
+     * Checking if a particular task is completed.
+     *
+     * @param s A string encoding whether a task is completed.
+     * @return Returns true if the task is completed, false otherwise.
+     */
+    private boolean isTaskDone(String s) {
+        return !s.equals("0");
+    }
+
+    /**
+     * Creates a file path.
+     *
+     * @param path The directory that to be created.
+     * @throws IOException
+     */
+    private void createFilePath(Path path) throws IOException {
+        //System.out.println(String.format("%s does not exist...", path.toString()));
+        Files.createDirectories(path);
+        //System.out.println(CREATE_FILEPATH);
```

Comment:
Should you remove comments that are not descriptions? These comments are from your old iteration of codes.

Reply:
yup thanks for spot